staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -418,7 +418,10 @@ type DeploymentStatus struct {
 	// +optional
 	AvailableReplicas int32 `json:"availableReplicas,omitempty" protobuf:"varint,4,opt,name=availableReplicas"`
 
-	// Total number of unavailable pods targeted by this deployment.
+	// Total number of unavailable pods targeted by this deployment. Essentially, this is the total
```

Comment:
would not recommend `Essentially,` here because it's redundant.  Can just be `This is the total ...`